compiler/quilt/tools/command.py

```diff
@@ -826,6 +826,12 @@ def install(package, hash=None, version=None, tag=None, force=False):
     store = PackageStore()
     existing_pkg = store.get_package(owner, pkg)
 
+    if existing_pkg is not None and not force:
+        print("{owner}/{pkg} already installed.".format(owner=owner, pkg=pkg))
+        overwrite = input("Overwrite? (y/n) ")
+        if overwrite.lower() != 'y':
+            return
+
```

Comment:
I seem to recall a UI issue with this... @akarve I think you didn't like this for some reason...

Reply:
The issue I want to avoid is this `quilt install usr/pkg --tag NON_EXISTENT_TAG` will prompt "y/N over-write?". I think it makes more sense for the install to fail first, instead of asking for permission for an over-write that will never happen.

Reply:
Ah... Makes sense